queue-manager/rango-preset/src/shared.ts

```diff
@@ -334,7 +334,7 @@ function mapSwapStepToPendingSwapStep(
 
 export function calculatePendingSwap(
   inputAmount: string,
-  bestRoute: BestRouteResponse,
+  bestRoute: Omit<BestRouteResponse, 'error' | 'errorCode' | 'traceId'>,
```

Comment:
Do you have any suggestion on changing underlying type in rango-types? @Ikari-Shinji-re

Reply:
You can use something like this. It helps avoid code duplication and more clearly conveys that the `bestRoute` is returned from the `ConfirmRouteResponse`:
```ts
NonNullable<ConfirmRouteResponse['result']>
```

Reply:
done